components/Common/UpperTitle.tsx

```diff
@@ -0,0 +1,23 @@
+type UpperTitleProps = {
+  title: string;
+};
+
+const UpperTitle = ({ title }: UpperTitleProps) => {
+  return (
+    <div className="flex flex-col-reverse md:flex-row md:items-center gap-1 md:gap-6">
+      <SectionLine />
+      <h4 className="h4 text-end md:text-right">{title}</h4>
+    </div>
+  );
+};
+
+export default UpperTitle;
+
+const SectionLine = () => {
```

Comment:
why do we need another component for this part?

Reply:
It is a private component, can't import it, can rewrite it inside the main component if needed

Reply:
@Drorka's PR solves that